v3/internal/commands/watcher.go

```
@@ -43,6 +47,30 @@ func Watcher(options *WatcherOptions) error {
 
 	ensureIgnored(&devconfig.Config.Ignore.File, "*_test.go")
 
+	// If we have application arguments, append them to the wails task
+	if options.AppArgs != "" {
+		for idx, execAction := range devconfig.Config.ExecStruct {
+			if execAction.Cmd == "wails3 task run" {
+				clonedExec := execAction
+
+				escapedAppArgs := options.AppArgs
```

golangci-lint (2.13.2)

[error] ineffectual assignment to escapedAppArgs

(ineffassign)